apps/patient/src/components/ReadyText.tsx

```diff
@@ -38,47 +34,66 @@ export const ReadyText = ({
   return null;
 };
 
-const roundUpTo15MinInterval = (pharmacyEstimatedReadyAt: Date): Dayjs => {
-  const estimatedReadyAtAsDayJs = dayjs(pharmacyEstimatedReadyAt);
-  const minutes = estimatedReadyAtAsDayJs.minute();
+const roundUpTo15MinInterval = (pharmacyEstimatedReadyAt: Date): Date => {
```

Comment:
thought it was weird this returns a dayjs instead of just a datetime, so changed that

Reply:
reverted, and added toDate